pepper-apis/src/main/java/org/broadinstitute/ddp/db/TransactionWrapper.java

```diff
@@ -33,6 +30,14 @@
  * are re-thrown after making an attempt to rollback the transaction.
  */
 public class TransactionWrapper {
+    // We are setting the VMs default timezone expectation that it will be same on server
+    // or at least specified in DB connection URL
+    // if timezones don't match, times and dates might be miscalculated/improperly converted by JDBC connector
+    // https://bugs.mysql.com/bug.php?id=91112
+    // Note 03/10/2020: Release of fixed Java connector is imminent and this might no longer be necessary
+    static {
+        TimeZone.setDefault(TimeZone.getTimeZone(ConfigManager.getInstance().getConfig().getString(ConfigFile.DEFAULT_TIMEZONE)));
+    }
```

Comment:
I settled on centralizing this change here rather than passing the timezone from everywhere in code that instantiates a TransactionWrapper . As comment indicates, I hope this is just temporary workaround.